dev-app/src/screens/CollectCardPaymentScreen.tsx

```diff
@@ -162,8 +170,8 @@ export default function CollectCardPaymentScreen() {
         },
       ],
     });
-    const paymentMethods = ['card_present'];
-    if (enableInterac) {
+    const paymentMethods = enabledPaymentMethodTypes;
+    if (enableInterac && !paymentMethods.includes('interac_present')) {
       paymentMethods.push('interac_present');
```

Comment:
Q: Should we push the same thing back to `enabledPaymentMethodTypes `?

Reply:
same as above (since the enableInterac is a separate UI component, i think its fine to have them separate)

Reply:
Got it, thanks!